docs/metatype.dev/blog/2024-09-26-introducing-grpc-runtime/index.mdx

```diff
@@ -0,0 +1,81 @@
+import BlogIntro from "@site/src/components/BlogIntro";
+
+# Introducing gRPC Runtime
+
+<BlogIntro text="We're excited to announce the new gRPC Runtime feature in Metatype, further enhancing our platform's ability to create versatile and powerful backends through typegraphs." />
+
+## What is Metatype?
+
+Before we dive into the gRPC Runtime, let's recap what Metatype is all about:
+
+Metatype is a revolutionary platform that allows developers to create typegraphs - strongly-typed, composable backend structures. With Metatype, you can:
+
+- Construct solid, type-safe backends
+- Leverage multiple runtimes for different services and protocols
+- Expose your typegraphs via GraphQL or HTTP endpoints
+
+## Introducing gRPC Runtime
+
+The new gRPC Runtime is the latest addition to Metatype's suite of runtimes, joining existing options like the HTTP runtime. This expansion allows you to incorporate gRPC services into your typegraphs, further enhancing the versatility of your Metatype-powered backends.
+
+## Key Technical Details
+
+### Architecture
+The gRPC Runtime integrates seamlessly with Metatype's existing architecture. It acts as a bridge between your typegraph and external gRPC services, allowing you to incorporate gRPC calls alongside other runtime operations in your backend logic.
+
+### Core Components
+
+1. **GrpcRuntime Class**: The main interface for defining gRPC interactions within your typegraph.
+2. **proto_file**: Path to the .proto file that defines the gRPC service.
+3. **endpoint**: The gRPC server address in the format `tcp://<host>:<port>`.
+4. **call method**: Creates a typegraph function for gRPC method calls.
+
+### Implementation
+
+Here's how the gRPC Runtime fits into a Metatype typegraph:
+
+```python
+from typegraph import Graph, Policy, typegraph
+from typegraph.graph.params import Cors
+from typegraph.runtimes.grpc import GrpcRuntime
+
+@typegraph(
+    cors=Cors(allow_origin=["https://metatype.dev", "http://localhost:3000"]),
+)
+def grpc(g: Graph):
```

Comment:
suggestion: rename the typegraph to avoid confusion for folks who've not worked with metatype before.